status/statusgo_backend/accounts/constants.nim

```diff
@@ -140,6 +140,7 @@ var NODE_CONFIG* = %* {
     }
   },
   "ShhextConfig": {
+    "EnableMailserverCycle": true,
```

Comment:
@richard-ramos I think that changes made here are not used in the context of app build from `base_bc` branch (cause it completely relays on "new" part of the `status-lib`), but this is not an issue, just a comment, in any case it does not affect anything.